DSCResources/cChocoPackageInstall/cChocoPackageInstall.psm1

```diff
@@ -389,41 +383,81 @@ Function Upgrade-Package {
     if ($cParams) {
         $chocoupgradeparams += " $cParams"
     }
-    $cmd = "choco upgrade $pName $chocoupgradeparams"
+    $cmd = "upgrade $pName $chocoupgradeparams"
     Write-Verbose -Message "Upgrade command: '$cmd'"

```

Comment:
I'd like to keep that Verbose message consistent from what it was previously - prepend it with 'choco'?

Reply:
done

Reply:
I don't see this change in the latest commit.

Reply:
Removed the choco